src/main/java/randoop/main/GenTests.java

```diff
@@ -496,9 +505,79 @@ public boolean handle(String[] args) throws RandoopTextuiException, RandoopInput
+   *
+   * <p>Class names are numbered with {@code basename} as the prefix. The package for tests is
+   * {@link GenInputsAbstract#junit_package_name}.
+   *
+   * @param junitCreator the {@link JUnitCreator} to create the test class source
+   * @param testSequences a list of {@link ExecutableSequence} objects for test methods
+   * @param codeWriter the {@link CodeWriter} to output the test classes
+   * @param basename the prefix for the class name
+   * @param testKind a {@code String} indicating the kind of tests for logging and error messages
+   */
+  private void writeTestFiles(
+      JUnitCreator junitCreator,
+      List<ExecutableSequence> testSequences,
+      CodeWriter codeWriter,
+      String basename,
+      String testKind) {
+    if (!testSequences.isEmpty()) {
+      if (GenInputsAbstract.progressdisplay) {
+        System.out.printf("%n%s test output:%n", testKind);
+        System.out.printf("%s test count: %d%n", testKind, testSequences.size());
+        System.out.printf("Writing JUnit tests...%n");
+      }
+      try {
+        List<File> testFiles = new ArrayList<>();
+
+        // Create and write test classes.
+        LinkedHashMap<String, CompilationUnit> testMap =
+            getTestASTMap(basename, testSequences, junitCreator);
+        for (Map.Entry<String, CompilationUnit> entry : testMap.entrySet()) {
+          String classname = entry.getKey();
+          String classSource = entry.getValue().toString();
+          testFiles.add(
+              codeWriter.writeClassCode(
+                  GenInputsAbstract.junit_package_name, classname, classSource));
+        }
+
+        // Create and write suite or driver class.
+        String driverName;
+        String classSource;
+        if (GenInputsAbstract.junit_reflection_allowed) {
+          driverName = basename;
+          classSource = junitCreator.createTestSuite(driverName, testMap.keySet());
+        } else {
+          driverName = basename + "Driver";
+          classSource = junitCreator.createTestDriver(driverName, testMap.keySet());
+        }
+        testFiles.add(
+            codeWriter.writeUnmodifiedClassCode(
+                GenInputsAbstract.junit_package_name, driverName, classSource));
+        if (GenInputsAbstract.progressdisplay) {
+          System.out.println();
+          for (File f : testFiles) {
+            System.out.printf("Created file: %s%n", f.getAbsolutePath());
+          }
+        }
+      } catch (RandoopOutputException e) {
+        System.out.printf(
+            "%nError writing " + testKind.toLowerCase() + " tests: " + e.getMessage());
+        System.exit(1);
+      }
+    } else {
+      if (GenInputsAbstract.progressdisplay) {
+        System.out.printf("%nNo " + testKind.toLowerCase() + " tests to output%n");
+      }
+    }
+  }
+
+  /**
+   * Creates a {@code plume.SimpleLog} that writes to standard output if {@code filename} is "-" (a
```

Comment:
It looks like there was a bad merge here.  Please remove this text and reinstate the proper Javadoc for this method.